_updates/2019-03-04-march-2019.md

```diff
@@ -0,0 +1,81 @@
+---
+title: March 2019
+---
+
+## Packages
+
+### [@sensenet/authentication-google@2.0.4](https://github.com/SenseNet/sn-client/releases/tag/%40sensenet%2Fauthentication-google%402.0.4)
+
+- Redundant type definition cleanup [#111](https://github.com/SenseNet/sn-client/pull/111)
+  
+### [@sensenet/authentication-jwt@1.0.8](https://github.com/SenseNet/sn-client/releases/tag/%40sensenet%2Fauthentication-jwt%401.0.8)
+
+- Redundant type definition cleanup [#111](https://github.com/SenseNet/sn-client/pull/111)
+
+### [@sensenet/client-core@1.4.1](https://github.com/SenseNet/sn-client/releases/tag/%40sensenet%2Fclient-core%401.4.1)
+
+- FormsAuthentication - fixed Logout error throwing on empty responses [#108](https://github.com/SenseNet/sn-client/pull/108)
+- FileUpload control [#82](https://github.com/SenseNet/sn-client/pull/82)
```

Comment:
```suggestion
- contentTypeName will be optional for upload requests [#82](https://github.com/SenseNet/sn-client/pull/82)
```
The FileUpload control sits in the control package and client-core not depends it, so I think we should be more specific in this case